tests/test_create_deprecated_class.py

```diff
@@ -0,0 +1,255 @@
+import inspect
```

Comment:
This is not anything new, I moved tests related to create_deprecated_class to a separate file.

Reply:
See also: https://github.com/scrapy/scrapy/issues/2194 :)